AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -16,6 +16,7 @@ class CallingContext: NSObject {
     // MARK: Properties
     private (set) var joinId: String!
     private (set) var displayName: String!
+    private (set) var isMuted: Bool = false
```

Comment:
I dont see we need this property right now. Remember me if I am wrong. 
We can use the call.isMute in CallingSDK 12.

Reply:
I'll make the appropriate changes after the main branch has been updated to use CallingSDK 12